models/comment.es6.js

```diff
@@ -1,36 +1,30 @@
 import Base from './base';
-import process from 'reddit-text-js';
 
 class Comment extends Base {
   _type = 'Comment';
 
   validators () {
-    const body = this.bodyValidator.bind(this);
+    const body_html = this.bodyHtmlValidator.bind(this);
```

Comment:
This should be camelCased instead - we minimize use of underscores to when we're dealing directly with the raw API results.